docs/Supported-Data-Types.md

```diff
@@ -0,0 +1,42 @@
+# Supported Data Types
+
+cBioPortal is a multimodal cancer data visualization tool and supports a variety of data types. For some data types, we have explicit support; for others, you can leverage generic assay or clinical data to assign arbitrary data to either a sample or patient. Beyond including data directly into cBioPortal, one can also link in additional external viewers specialized for particular data types, or even entirely different dashboards or portals. This combination of structured and flexible data formats, as well as the option to link out to other viewers, allows cBioPortal to be used for many different combinations of clinical data and profiling methods.
+
+## Molecular Data
+
+| Assay | Processed Data | cBioPortal Data Type | Example cBioPortal Study |
+| --- | --- | --- | --- |
+| Bulk DNA Sequencing | Mutations | [Mutations](./File-Formats.md#mutation-data) | [MSK-IMPACT (Nat Med, 2017)](https://www.cbioportal.org/study/summary?id=msk_impact_2017) |
```

Comment:
and Derived Data

Reply:
Not sure I fully follow: some derived data is listed like mutational signatures and Genetic Ancestry. Each of the rows below with `-` are based on Bulk DNA Sequencing. Do you mean additional derived data on top of those?